src/index.html

```diff
@@ -7,21 +7,377 @@
+                  class="nav__link"
+                >
+                  Tablets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__content">
+          <div
+            class="card"
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating rating">
+              <div class="rating__stars stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="rating__reviews">Reviews: 5</p>
+            </div>
+            <p class="card__price price">
+              <span class="price__text">Price:</span>
+              <span class="price__amount">$2,199</span>
+            </p>
+
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              BUY
+            </a>
+          </div>
+          <div class="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating rating">
+              <div class="rating__stars stars">
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star stars__star--active"></div>
+                <div class="stars__star"></div>
+              </div>
+              <div class="rating__reviews">Reviews: 5</div>
+            </div>
+            <div class="card__price price">
+              <div class="price__text">Price:</div>
+              <div class="price__amount">$2,199</div>
+            </div>
```

Comment:
```suggestion
            <div class="card__rating rating">
              <div class="rating__stars stars stars--4">
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
                <div class="stars__star"></div>
              </div>
              <p class="rating__reviews">Reviews: 5</p>
            </div>
            <p class="card__price price">
              <span class="price__text">Price:</span>
              <span class="price__amount">$2,199</span>
            </p>
```
Consider fixing same issues for your blocks